src/Utilities/CodeMetrics/CodeAnalysisMetricData.AssemblyMetricData.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
is this right copyright header?

Reply:
I used the same header as other files in the repo, for example https://github.com/dotnet/roslyn-analyzers/blob/master/src/Utilities/DiagnosticHelpers.cs#L1